source/FreeRTOS_Routing.c

```diff
@@ -1504,7 +1504,7 @@ const char * pcEndpointName( const NetworkEndPoint_t * pxEndPoint,
                     ( void ) FreeRTOS_inet_ntop( FREERTOS_AF_INET4,
                                                  ( const void * ) &( pxEndPoint->ipv4_settings.ulIPAddress ),
                                                  pcBuffer,
-                                                 uxSize );
+                                                 ( uint32_t ) uxSize );
```

Comment:
Why is `uxSize` casted to `uint32_t`, can we use the `socklen_t` which is what is expected by `FreeRTOS_inet_ntop`. Though `socklen_t` is `uint32_t` it might be more cleaner and portable if type of `socklen_t` is changed.

Reply:
Done